backend/src/main/kotlin/me/andrewda/handlers/ItemHandler.kt

```diff
@@ -9,20 +9,21 @@ import io.ktor.routing.patch
 import io.ktor.routing.post
 import me.andrewda.controllers.ItemController
 import me.andrewda.models.NewItem
+import me.andrewda.utils.getApiResponse
 import me.andrewda.utils.respond
 
 fun Route.item() {
     get("/items") {
         val items = ItemController.findAll()
-        call.respond(items.map { it.api })
+        call.respond(items.map { it.getApiResponse() })
```

Comment:
Would it be possible to make this `apiResponse()` instead of `getApiResponse()`? That way, it looks more like we're accessing a property, like we were previously.

Reply:
Potentially. Not exactly sure what the standard is here (but we should conform to whatever that is) @jamdan2